===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/menus/desktop-file-utils/src/update-desktop-database.c,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- src/update-desktop-database.c	2004/09/08 15:43:14	1.6
+++ src/update-desktop-database.c	2005/01/10 00:43:48	1.7
@@ -449,6 +449,8 @@
   for (i = 0; data_dirs[i] != NULL; i++)
     args[i] = g_build_filename (data_dirs[i], "applications", NULL);
 
+  args[i] = NULL;
+
   g_strfreev (data_dirs);
 
   return (const char **) args;
